src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -36,14 +36,14 @@
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
```

Comment:
```suggestion
    public static final String MESSAGE_SUCCESS = "New candidate added: %1$s";
    public static final String MESSAGE_DUPLICATE_PERSON = "This candidate already exists in the address book";
```

Reply:
Perhaps the `person` on these two lines should be changed to `candidate`.

Reply:
nice catch! The String could also be changed to MESSAGE_DUPLICATE_CANDIDATE.

Reply:
You are right Leon! How did I missed that 🤣